packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -476,8 +480,11 @@ export class ColorPicker
 
     let inputValue: string;
 
-    if (this.isClearable && !input.value) {
+    if (!input.value) {
       inputValue = "";
+      this.channelInputClear = true;
+      // reset this to allow typing in new value when channel input is empty
```

Comment:
question: reading this comment, would a private property like `allowTypingNewVal` be better to allow typing a new value when the input is empty?

Maybe a single property like `isInputEmpty` or `isClear` could be used in place of `channelInputClear` and `arrowUpOrDownTracker`?

Reply:
I'm clearing the `arrowUpOrDownTracker` to prevent an edge case [here](https://github.com/Esri/calcite-design-system/pull/9701/files#diff-ab28035ea6ab07b366c5635f72ec60a722bc674f8fe0a6aeb80c8d43ef4bbb7aR584). If arrowUp or arrowDown are triggered before clearing the input, and then a new value is typed, the logic within that `if` will be executed. I can try to validate this with a single variable, like you suggested.